tools/hsmtool.c

```diff
@@ -368,6 +371,55 @@ static int guess_to_remote(const char *address, struct node_id *node_id,
 	return 1;
 }
 
+static int dumponchaindescriptors(const char *hsm_secret_path, const char *passwd)
+{
+	struct secret hsm_secret;
+	u8 bip32_seed[BIP32_ENTROPY_LEN_256];
+	u32 salt = 0;
+	struct ext_key master_extkey;
+	char *enc_xpub, *descriptor, checksum[DESCRIPTOR_CHECKSUM_LENGTH + 1];
+
+	if (passwd)
+		get_encrypted_hsm_secret(&hsm_secret, hsm_secret_path, passwd);
+	else
+		get_hsm_secret(&hsm_secret, hsm_secret_path);
+
+	/* We use m/0/0/k as the derivation tree for onchain funds. */
```

Comment:
Needs reference. Where in the codebase do we do this?

Reply:
One of these comments is wrong then 

>  	 *  - m/iH/0/k corresponds to the k'th keypair of the external chain of 
 	 * account number i of the HDW derived from master m. 

is not the same is.

>  We use m/0/0/k as the derivation tree for onchain funds. 

For the record, the intention here is that you'd update the code comment so other readers know where you're getting the info from, not just post a reply in github.